### ISSUE TITLE
time template func broken

### ISSUE THREAD

**Issue by bep** [spf13/hugo#3489]
Som of the theme demo sites now fails:

```
Started building sites ...
ERROR 2017/05/19 21:20:57 error processing shortcode "theme/shortcodes/attachments.html" for page "shortcode/attachments.md": template: theme/shortcodes/attachments.html:18:40: executing "theme/shortcodes/attachments.html" at <time>: wrong number of args for time: want 0 got 1
ERROR 2017/05/19 21:20:57 error processing shortcode "theme/shortcodes/attachments.html" for page "shortcode/attachments.md": template: theme/shortcodes/attachments.html:11:40: executing "theme/shortcodes/attachments.html" at <time>: wrong number of args for time: want 0 got 1
```

/cc @moorereason

**Comment by moorereason** [comment]
Fixed in #3451

**Comment by bep** [comment]
Ah, my memory fades. Thanks.
